### DISCUSSION TITLE
在form中如何取到namelist中相同的对象名下的所有字段

### DISCUSSION THREAD

**Discussion by noob001rightnow** [ant-design/ant-design#42210] [Q&A]
我设置了类似['nameObj', 'name'] , ['nameObj', 'age'], ['nameObj', 'address']，['second', 'name']，['second', 'age'] 的formitem，如何在校验的时候只检验nameObj下的所有字段，或者有没有取巧的方法。因为nameObj下可能有十几个字段，不会要一个一个写出来吧（苦笑）

**Comment by zombieJ** [comment]
嗯，感觉可以的。
